pkg/asset/manifests/openshift.go

```diff
@@ -3,8 +3,10 @@ package manifests
 import (
 	"context"
 	"encoding/base64"
+	"io/ioutil"
```

Comment:
For new code, I would use `os` library instead of iotuil as it's deprecated in recent golang version
```
Deprecation of io/ioutil
The io/ioutil package has turned out to be a poorly defined and hard to understand collection of things. All functionality provided by the package has been moved to other packages. The io/ioutil package remains and will continue to work as before, but we encourage new code to use the new definitions in the io and os packages. Here is a list of the new locations of the names exported by io/ioutil:

Discard => io.Discard
NopCloser => io.NopCloser
ReadAll => io.ReadAll
ReadDir => os.ReadDir (note: returns a slice of os.DirEntry rather than a slice of fs.FileInfo)
ReadFile => os.ReadFile
TempDir => os.MkdirTemp
TempFile => os.CreateTemp
WriteFile => os.WriteFile
```

Reply:
Thanks for the note!
The problem is that the installer is 1.14 golang, so os doesn't have that function on the installer golang version :/
I guess once the installer will move to 1.16 this will need to be fixed everywhere